charts/apiclarity/values.yaml

```diff
@@ -81,3 +73,56 @@ apiclarity-postgresql:
 
 # End of APIClarity Postgres Values
 #######################################################################################
+
+#######################################################################################
+## APIClarity Traffic Source Values
+
+trafficSource:
+  envoyWasm:
+    enabled: false
+
+    ## Enable Istio Check Pre-Install Job
+    ##
+    enableIstioVerify: true
+
+    ## Enable APIClarity WASM filter in the following namespaces
+    ##
+    namespaces:
+      - default
+
+  kong:
+    enabled: false
+    ## Specify the name of the proxy container in Kong gateway to patch
+    ##
+    containerName: "proxy"
+    ## Specify the name of the Kong gateway deployment to patch
+    ##
+    deplyomentName: ""
+    ## Specify the namespace of the Kong gateway deployment to patch
+    ##
+    deplyomentNamespace: ""
+    ## Specify the name of the ingress resource to patch
+    ##
+    ingressName: ""
+    ## Specify the namespace of the ingress resource to patch
+    ##
+    ingressNamespace: ""
+
+  tyk:
+    enabled: false
+
+    ## Enable Tyk Check Pre-Install Job
```

Comment:
I think that it's better to comment also on the "enabled" to avoid confusion about what comment is it.

Also, maybe: Enable Tyk verification in a Pre-Install Job?